.github/workflows/security-scanners.yml

```
@@ -0,0 +1,22 @@
+name: Security scanners
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  run-scanners:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Ruby and install gems
+        uses: ruby/setup-ruby@473e4d8fe5dd94ee328fdfca9f8c9c7afc9dae5e
+        with:
+          bundler-cache: true
+
+      - name: Run security checks
+        run: |
+          bin/bundle exec brakeman -q --exit-on-warn
```

Ich fände es noch gut, wenn wir da auch immer die aktuelle Version verwenden.

```suggestion
          bin/bundle exec brakeman -q --exit-on-warn --ensure-latest --ensure-ignore-notes
```

---

Finde ich gut. Das macht dann zwar immer das CI rot sobald Brakeman ein Update rausbringt. Somit ist unser CI-Success abhängig von externen Faktoren, was mir nicht ganz so gut gefällt. Aber immerhin ist es ein Ansporn, Brakeman aktuell zu halten.